components/Snackbar/src/MDCSnackbarMessage.h

```diff
@@ -143,6 +143,11 @@ extern NSString * __nonnull const MDCSnackbarMessageBoldAttributeName;
  */
 @property(nonatomic, readonly, nullable) NSString *voiceNotificationText;
 
+/**
+ Use the older legacy version of snackbar. Default is YES.
+ */
+@property(nonatomic, assign) BOOL usesLegacySnackbar;
```

Comment:
Please consider making this (or adding) a class property so it may be set once for clients who want to adopt the new style.

Reply:
ack, done